template/config/core/main.ts

```diff
@@ -4,10 +4,13 @@ import router from '@/config/router';
 import store from '@/store';
 import '@/config/register-service-worker';
 import '@/theme/_all.scss';
+/* tslint:disable-next-line */
```

Comment:
@ttaylor92 for all these tslint things, please update the tslint.json file with all the rules necessary, I suggest removing these inline comments and adding these updated rules to the tslint and tsconfig json file.

Reply:
change made